### PR TITLE
[TACHYON-641] Redirect BlockWorker.getStoreMeta() calls to BlockDataM…

### PR DESCRIPTION
**Pull request by miasantreble**
…anager

### CODE REVIEW

**Comment by calvinjia** [comment]
+1 LGTM

File: servers/src/main/java/tachyon/worker/block/BlockWorker.java [hunk 1]
```diff
@@ -96,12 +96,12 @@ public BlockWorker(TachyonConf tachyonConf) throws IOException {
     // Setup metrics collection
     WorkerSource workerSource = new WorkerSource();
     mWorkerMetricsSystem = new MetricsSystem("worker", mTachyonConf);
-    workerSource.registerGauges(this);
-    mWorkerMetricsSystem.registerSource(workerSource);
-
     // Set up BlockDataManager
     mBlockDataManager = new BlockDataManager(tachyonConf, workerSource);
 
+    workerSource.registerGauges(mBlockDataManager);
```
Comment (apc999) [inline servers/src/main/java/tachyon/worker/block/BlockWorker.java]: Please also move line 96,97,98 down, to the line right before `workerSource.registerGauges(mBlockDataManager);`, as they all belong to "Setup metrics collection" and we don't want to separate them

Comment (miasantreble) [inline servers/src/main/java/tachyon/worker/block/BlockWorker.java]: done :sweat_smile:

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**